SwanContents/swancontents/js/utils.js

```diff
@@ -0,0 +1,21 @@
+define([
+    'base/js/utils'
+], function (utils) {
+
+    // Wrap upstream promising_ajax to append the authorization header
```

Comment:
Can the commit message explain a bit more why we now put this token in the authorization header? What is this token? What are the requests that now will have this token? Why wasn't this there before?